src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+body,
+header,
+nav,
+ul {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Move this to `reset.scss` file. Also, you already set `box-sizing: border-box` under * selector, so you don't need to set this here again